libraries/AP_Motors/AP_MotorsTri.cpp

```diff
@@ -374,3 +424,43 @@ bool AP_MotorsTri::arming_checks(size_t buflen, char *buffer) const
     // run base class checks
     return AP_MotorsMulticopter::arming_checks(buflen, buffer);
 }
+
+// Get the testing order for the motors
+uint8_t AP_MotorsTri::get_motor_test_order(uint8_t i)
```

Comment:
You can gate this behind `#if APM_BUILD_TYPE(APM_BUILD_UNKNOWN)` so we can be sure that its only there in the example.

Reply:
Nice, I like that. Should I do the same thing to `get_pitch_factor`? Nothing seems to use it yet but could be useful in the future.

Reply:
`get_pitch_factor` is used by turtle mode in copter I think. I'm not sure if it would handle tri correctly tho.

Reply:
I'll go ahead and gate it too then. Someone can always ungate it in the future.